docs/EN/operate/build.md

```diff
@@ -9,8 +9,8 @@ Update with [GitHub](#update-trio-with-github) | [Mac-Xcode](#update-trio-with-x
 The Trio repository contains instructions for building the Trio app using a browser and GitHub. More detailed instructions are also available in LoopDocs.
 
 * [GitHub Build Instructions](https://github.com/nightscout/Trio/blob/dev/fastlane/testflight.md)
-* [LoopDocs: GitHub Build First-Time](https://loopkit.github.io/loopdocs/gh-actions/gh-first-time/)
-* [LoopDocs: GitHub Build Other Apps](https://loopkit.github.io/loopdocs/gh-actions/gh-other-apps/)
+* [LoopDocs: GitHub Build First-Time](https://loopkit.github.io/loopdocs/browser/intro-summary/)
```

Comment:
I think this is a better link.
`https://loopkit.github.io/loopdocs/browser/bb-overview/`

Reply:
Agreed. PR is updated.